users/src/main/java/io/crate/operation/user/UserManagerServiceFactory.java

```diff
@@ -35,9 +39,22 @@ public UserManager create(Settings settings,
                               ThreadPool threadPool,
                               ActionFilters actionFilters,
                               IndexNameExpressionResolver indexNameExpressionResolver,
-                              SysSchemaInfo sysSchemaInfo) {
+                              SysSchemaInfo sysSchemaInfo,
+                              SystemCollectSource systemCollectSource) {
         TransportCreateUserAction transportAction = new TransportCreateUserAction(
             settings, transportService, clusterService, threadPool, actionFilters, indexNameExpressionResolver);
-        return new UserManagerService(transportAction);
+        UserManagerService userManagerService = new UserManagerService(transportAction);
+        registerUsersTable(sysSchemaInfo, clusterService, userManagerService, systemCollectSource);
+        return userManagerService;
+    }
+
+    private void registerUsersTable(SysSchemaInfo sysSchemaInfo, ClusterService clusterService,
+                                    UserManagerService usersManagerService, SystemCollectSource systemCollectSource) {
+        sysSchemaInfo.registerSysTable(SysUsersTableInfo.IDENT.name(), new SysUsersTableInfo(clusterService));
```

Comment:
maybe we could really have a SysTableRegistry that encapsulates this logic here?